packages/terra-form-textarea/src/Textarea.jsx

```diff
@@ -79,6 +79,10 @@ const propTypes = {
    * functionality such as managing focus, selection, or animations.
    */
   refCallback: PropTypes.func,
+  /**
+   * Value to set for the aria-label.
+   */
+  label: PropTypes.string.isRequired,
```

Comment:
Adding a required prop will be a breaking change, this will require a major version bump

Reply:
Thoughts on making this prop `ariaLabel`?

Reply:
Let's make this non-required and change the prop to ariaLabel. If people use the TextareaField, we require a label for that component and map it to a label element so the aria-label on the textarea wouldn't be needed. Its only if you render the `<Textarea />` without our `<Field />` composition that the textarea needs to be labelled.

Reply:
fixed here https://github.com/cerner/terra-core/pull/2583/commits/cba49d48d985f385a4e210fc459814560d02c3b2.

Reply:
@bjankord 

> Its only if you render the <Textarea /> without our <Field /> composition that the textarea needs to be labelled.

If accessibility is violated without it, then shouldn't it be a required prop?

Reply:
@nramamurth You are correct, when using `<Textarea />` without a label would be an accessibility violation. In a future release, we can make it a required prop, at this time though, we don't want to introduce a breaking change to this package.  

One piece of complexity to work out with making `ariaLabel` a required prop is around a type compatibility issue with the `<TextareaField />` `label` prop. The TexareaField component has [a required `label` prop, however it is set to a node type.](https://github.com/cerner/terra-core/blob/master/packages/terra-form-textarea/src/TextareaField.jsx#L15), where as for `ariaLabel` on the Textarea, we want/need to use a string type. 

If we make `<Texarea />` have a required `ariaLabel`, we need to find some way to get the `label` from `<TextareaField label={<span>Test label</span>}>` down to the `<Textarea />` `ariaLabel` prop. The solution for this is likely [introduce a breaking change](https://github.com/cerner/terra-core/issues/2294) for `<TextareaField />` to take a string for the `label` prop instead of a node so we can cleanly pass that string down to the `<Textarea />` `ariaLabel` prop. 

We'll also want to test with screen readers and see how it reads `<TextareaField label="Test label" />` when that component has both a `<label />` element mapping to the `textarea` and an `ariaLabel` on the `textarea` element.